packages/sdk/governance/src/proposals.ts

```diff
@@ -206,7 +206,10 @@ export class ProposalBuilder {
 
   fromJsonTx = async (tx: ProposalTransactionJSON): Promise<ProposalTransaction> => {
     // handle sending value to unregistered contracts
-    if (!RegisteredContracts.includes(tx.contract)) {
+    if (
+      !RegisteredContracts.includes(tx.contract.toString().replace('Proxy', '') as CeloContract) &&
```

Comment:
nit:  Consider creating helper functions to deal with the "Proxy" suffix.

I see that the "Proxy" string is also hard coded here:  https://github.com/celo-org/celo-monorepo/blob/master/packages/sdk/governance/src/proposals.ts#L232